dataflow/conftest.py

```diff
@@ -525,6 +555,46 @@ def dataflow_flex_template_run(
 
         Utils.dataflow_jobs_cancel(job_id)
 
+    @staticmethod
+    def dataflow_extensible_template_run(
```

Comment:
I think the "official" name for the older (non-Flex) templates is "Classic Templates". Can we rename this to `dataflow_classic_template_run`? Or was there a reason why "extensible"?

https://cloud.google.com/dataflow/docs/guides/templates/creating-templates